static/src/javascripts/projects/common/views/acquisitions-epic-testimonials.html

```diff
@@ -4,7 +4,7 @@ <h2 class="contributions__title contributions__title--epic">
             Since you’re here &hellip;
         </h2>
         <p class="contributions__paragraph contributions__paragraph--epic">
-            &hellip; we’ve got a small favour to ask. More people are reading the Guardian than ever, but far fewer are paying for it. Advertising revenues across the media are falling fast. And <span class="contributions__highlight">unlike many news organisations, we haven’t put up a paywall – we want to keep our journalism as open as we can</span>. So you can see why we need to ask for your help. The Guardian’s independent, investigative journalism takes a lot of time, money and hard work to produce. But we do it because we believe our perspective matters – because it might well be your perspective, too.
+            <%=p1%>
```

Comment:
You're gonna need to update the other test that uses this template to pass this in as well, right?